axelrod/tests/unit/test_match.py

```diff
@@ -360,16 +354,17 @@ def test_sparklines(self):
 class TestSampleLength(unittest.TestCase):
     def test_sample_length(self):
         for seed, prob_end, expected_length in [
-            (0, 0.5, 3),
+            (0, 0.5, 2),
             (1, 0.5, 1),
-            (2, 0.6, 4),
-            (3, 0.4, 1),
+            (2, 0.6, 1),
+            (3, 0.4, 2),
         ]:
-            axl.seed(seed)
-            self.assertEqual(axl.match.sample_length(prob_end), expected_length)
+            rg = RandomGenerator(seed)
+            r = rg.random()
```

Comment:
```suggestion
            random = RandomGenerator(seed)
            r = random.random()
```

To be consistent with changes elsewhere? 

I'd actually suggest `rag = RandomerGenerator(seed)` and using that elsewhere. 

Not at all vital to change: whatever you prefer @marcharper.

Reply:
Sure. I've been using the `RandomGenerator` object as a drop-in replacement for the standard `random` module, but it's not quite the same.

Reply:
I changed it to `rng` for now. We can decide on a universal approach throughout pending your documentation effort.